components/add_groups_to_channel_modal/add_groups_to_channel_modal.jsx

```diff
@@ -22,6 +22,7 @@ export default class AddGroupsToChannelModal extends React.PureComponent {
     static propTypes = {
         currentChannelName: PropTypes.string.isRequired,
         currentChannelId: PropTypes.string.isRequired,
+        teamID: PropTypes.string.isRequired,
```

Comment:
nit: `teamId`? not sure what the new convention is anymore 🤷‍♂️

Reply:
Not sure, I see some other instances too.